src/commands/pickFuncProcess.ts

```diff
@@ -106,6 +105,26 @@ async function startFuncTask(funcTask: Task, timeoutInSeconds: number, timeoutEr
     return await waitForStartPromise;
 }
 
+/**
+ * Gets the innermost child pid for a unix process. This is only really necessary if the user installs 'func' with a tool like 'npm', which uses a wrapper around the main func exe
+ */
+async function getInnermostUnixPid(pid: string): Promise<string> {
+    return await new Promise<string>((resolve: (pid: string) => void, reject: (e: Error) => void): void => {
+        unixPsTree(parseInt(pid), (error: Error | undefined, children: unixPsTree.PS[]) => {
+            if (error) {
+                reject(error);
+            } else {
+                const child: unixPsTree.PS | undefined = children.pop();
```

Comment:
Would this get the innermost?  Isn't this only getting the first child?

Reply:
I mean 'pop' is removing the last element from the array. I'm assuming the last element is the innermost child, but I suppose it might not be. That being said - I'm not sure how I can test that since I can only reproduce a case where 'func' is the first (and only) child.

Let me know if you think it's worth it and I'll try to repro a case with multiple nested children

Reply:
Nah, I think if it becomes a problem, we can resolve it then.